### DISCUSSION TITLE
Regular Expressions help

### DISCUSSION THREAD

**Discussion by dpente** [otago-polytechnic-bit-courses/ID607001-intro-app-dev-concepts#18] [General]
Hey guys if you would like some info on regular expressions (which are needed for some of our formative assessment questions) Hamza had us use this site last semester in osc. 

https://regexone.com/

Its great as it is completely interactive and will take you through things step by step.

**Comment by Grayson-Orr** [comment]
Thanks for that @dpente. This is a great resource.
